examples/gui_qt_asyncio.py

```diff
@@ -34,13 +52,8 @@ def __init__(self):
         self.canvas = WgpuWidget(splitter)
         self.output = QtWidgets.QTextEdit(splitter)
 
-        # With QtAsyncio, the callbacks can now return a future. You'd
-        # think that you could also return a coro, but we need to wrap
-        # it into a future, making this code a bit ugly.
-        # self.button.clicked.connect(self.whenButtonClicked)  # why not :/
-        self.button.clicked.connect(
-            lambda: asyncio.ensure_future(self.whenButtonClicked())
-        )
+        # self.button.clicked.connect(self.whenButtonClicked)  # see above :(
+        async_connect(self.button.clicked, self.whenButtonClicked)
```

Comment:
Just wondering how this handles arguments for the event handler?

Reply:
As it is implemente now, it is assumed there are no args. But since this is just an example, that seems fine, I think.